packages/lesswrong/lib/editor/make_editable.js

```diff
@@ -45,6 +46,17 @@ export const makeEditable = ({collection, options = {}}) => {
     formGroup,
     permissions,
     fieldName = "",
+    hintText = <div>
+      <div>
+        Write your {collection.options.singleResolverName} here. Select text for formatting options.
```

Comment:
On the user-profile page, the Moderation Guidelines editor now has the placeholder text: "Write your user here."